main.rb

```diff
@@ -0,0 +1,53 @@
+require_relative 'app'
+# rubocop:disable Metrics/CyclomaticComplexity
+# rubocop:disable Metrics/MethodLength
```

Comment:
- [ ] As required by Microverse, you are not meant to use any linters disable rule on your codebase. I suggest you remove this and try to fix the issue  :+1: :100: You can consider using an if statement inside of a loop. :+1: :smile: